compiler-cli/src/hex/auth.rs

```diff
@@ -135,16 +135,28 @@ encrypt your Hex API key.
     }
 }
 
+fn get_env_password() -> std::result::Result<String, VarError> {
+    std::env::var(PASS_ENV_NAME).inspect(|_| {
+        println!("Using {PASS_ENV_NAME} environment variable, but note it will be deprecated in favour of {API_ENV_NAME}");
```

Comment:
It has been deprecated, not it will be in future!

Could you merge the 2 functions back into one please 🙏

Reply:
with pleasure! 

changed the message to simply "deprecated", see my [comment below](https://github.com/gleam-lang/gleam/pull/4336#issuecomment-2749391246). And merged the 2 functions back the way they were.